play_tester.py

```diff
@@ -25,5 +25,5 @@
 #pp.pprint(clean_wave_4_data())
 
 # Wave 05 user data
-#print("\n-----Wave 05 user_data-----")
-#pp.pprint(clean_wave_5_data())
+print("\n-----Wave 05 user_data-----")
+pp.pprint(clean_wave_5_data())
```

Comment:
Would advise to omit or comment out print statements in your final submission! In large projects, the console is great for debugging but when it's used in production or by other developers who are unfamiliar with your code, an excess of printing can clutter and confuse them as they look at the console raising questions like “What does this mean, why is it printed, where is this coming from?”